frontend-web/src/app/api/auth/login/route.ts

```diff
@@ -0,0 +1,41 @@
+import { cookies } from 'next/headers';
+
+export async function POST(req: Request): Promise<Response> {
+  const { email, password } = await req.json();
+
+  try {
+    const response = await fetch('http://localhost:3000/api/auth/login', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ email, password }),
+    });
+
+    if (!response.ok) {
+      const { message } = await response.json();
+
+      return new Response(JSON.stringify({ message }), { status: 401 });
+    }
+
+    const { token, user } = await response.json();
+
+    (await cookies()).set({
+      name: 'token',
```

Comment:
mettre le nom du token dans une constante

```suggestion
      name: SPORT_APP_AUTH_TOKEN,
```